src/analyses/basePriv.ml

```diff
@@ -263,13 +272,8 @@ struct
   let read_global ask getg (st: BaseComponents (D).t) x =
     if is_unprotected ask x then (
       let get_mutex_global_x = get_mutex_global_x_with_mutex_inits getg x in
-      (* None is VD.top () *)
-      match CPA.find_opt x st.cpa, get_mutex_global_x with
-      | Some v1, Some v2 -> VD.meet v1 v2
-      | Some v, None
-      | None, Some v -> v
-      | None, None -> VD.bot () (* Except if both None, needed for 09/07 kernel_list_rc *)
```

Comment:
Interesting that this isn't needed, because it must've been at some point. I think this was doing the "long" meet like `lock` does, but just for a single variable.

Reply:
I think it is not even an issue of it being spurious, but there actually was a case where it was unsound, as the comment indicates. Maybe when @stilscher  is done with verifying the side-effecting TD3, we could ask her to verify all our fabulous privatizations to avoid such issues :rofl: